README.md

```
@@ -190,6 +190,10 @@ Collection returned on endpoint: /api/logs/withCollection, size: 2, status: 200
 
 ### Changelog:
 
+##### 27.01.2026
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)